Core/include/Acts/Seeding/SeedFinderUtils.hpp

```diff
@@ -65,6 +66,22 @@ void transformCoordinates(std::vector<external_spacepoint_t*>& vec,
                           external_spacepoint_t& spM, bool bottom,
                           std::vector<LinCircle>& linCircleVec,
                           callable_t&& extractFunction);
+
+/// @brief Check the compatibility of spacepoint coordinates in xyz assuming the Bottom-Middle direction with the strip meassument details
+///
+/// @tparam external_spacepoint_t The external spacepoint type.
+/// @tparam sp_range_t container type for the seed point collections.
+///
+/// @param[in] m_config seedfinder config containing the delegate to the strip meassument details
+/// @param[in] spacepointPosition Spacepoint coordinates in xyz plane.
+/// @param[in] toleranceParam Parameter used to evaluate if spacepointPosition is inside the detector elements.
+/// @param[out] outputCoordinates The output vector to write to.
+/// @returns bool that says if spacepoint is conpatible with being inside the detector element.
```

Comment:
```suggestion
/// @returns bool that says if spacepoint is compatible with being inside the detector element.
```

Reply:
I don't understand this sentence - how would a SpacePoint not correspond to a measurement?

Reply:
This is what I explained here: https://github.com/acts-project/acts/pull/1255#discussion_r874475713

If the seed direction is compatible with the channels fired, the candidate space points move to the next checks of seed production. If, instead, the seed direction requires the space points to "move" outside the channel length, i.e. outside the detector element, the combination of the three space points is discarded, reducing the quantity of low quality seeds.